hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -29,6 +29,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
+import javafx.util.Pair;
```

Comment:
It seems `javafx.util` is not available in Java 11 by default.  Can we use the one from `org.apache.commons.lang3.tuple` instead?

Reply:
Done. Thanks @adoroszlai.